types/types.go

```diff
@@ -82,21 +82,21 @@ const (
 
 // RuleWithContent structure with rule and rule content
 type RuleWithContent struct {
-	Module          types.RuleID    `json:"module"`
-	Name            string          `json:"name"`
-	Summary         string          `json:"summary"`
-	Reason          string          `json:"reason"`
-	Resolution      string          `json:"resolution"`
-	MoreInfo        json.RawMessage `json:"more_info"`
-	ErrorKey        types.ErrorKey  `json:"error_key"`
-	Condition       string          `json:"condition"`
-	Description     string          `json:"description"`
-	TotalRisk       int             `json:"total_risk"`
-	RiskOfChange    int             `json:"risk_of_change"`
-	PublishDate     time.Time       `json:"publish_date"`
-	Active          bool            `json:"active"`
-	Internal        bool            `json:"internal"`
-	Generic         string          `json:"generic"`
-	Tags            []string        `json:"tags"`
+	Module          types.RuleID   `json:"module"`
+	Name            string         `json:"name"`
+	Summary         string         `json:"summary"`
+	Reason          string         `json:"reason"`
+	Resolution      string         `json:"resolution"`
+	MoreInfo        string         `json:"more_info"`
```

Comment:
are you sure it won't break markdown parsing?

Reply:
I don't think so. Apart of that, more info files cannot be a `json.RawMessage` because more_info.md files doesn't contain any json at all, so it was failing at marshalling the response when MoreInfo contains some markdown